<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="TanmayJ13401" name="lm.lmSolution.PlaceOrderTests" tests="12" failures="0" timestamp="22 Jan 2019 07:12:29 GMT" time="15.871" errors="1">
  <testcase name="Testcase_placeOrder_verifyDistancesCount" time="1.230" classname="lm.lmSolution.PlaceOrderTests"/>
  <testcase name="Testcase_placeOrder_verifyID" time="1.247" classname="lm.lmSolution.PlaceOrderTests"/>
  <testcase name="Testcase_placeAdvanceOrder_verifyDistancesCount" time="1.221" classname="lm.lmSolution.PlaceOrderTests"/>
  <testcase name="Testcase_placeOrder_verifyCurrency" time="1.302" classname="lm.lmSolution.PlaceOrderTests"/>
  <testcase name="Testcase_placeOrder_verifySingleStopValue" time="1.014" classname="lm.lmSolution.PlaceOrderTests"/>
  <testcase name="Testcase_placeOrder_verifyFareAboveTwoKms" time="1.247" classname="lm.lmSolution.PlaceOrderTests"/>
  <testcase name="Testcase_placeAdvanceOrder_VerifyId" time="2.708" classname="lm.lmSolution.PlaceOrderTests"/>
  <testcase name="Testcase_placeOrder_verifyFareBelowTwoKms" time="1.182" classname="lm.lmSolution.PlaceOrderTests"/>
  <testcase name="Testcase_placeAdvanceOrder_verifyFareBelowTwoKms" time="1.244" classname="lm.lmSolution.PlaceOrderTests"/>
  <testcase name="Testcase_placeAdvanceOrder_verifyFareAboveTwoKms_NightCharges" time="1.022" classname="lm.lmSolution.PlaceOrderTests">
    <error type="java.lang.AssertionError" message="JSONObject[&quot;drivingDistancesInMeters&quot;] not found.">
      <![CDATA[java.lang.AssertionError: JSONObject["drivingDistancesInMeters"] not found.
	at org.testng.Assert.fail(Assert.java:94)
	at lm.lmSolution.PlaceOrderTests.Testcase_placeAdvanceOrder_verifyFareAboveTwoKms_NightCharges(PlaceOrderTests.java:386)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- Testcase_placeAdvanceOrder_verifyFareAboveTwoKms_NightCharges -->
  <testcase name="Testcase_placeAdvanceOrder_verifyFareBelowTwoKms_NightCharges" time="1.216" classname="lm.lmSolution.PlaceOrderTests"/>
  <testcase name="Testcase_placeAdvanceOrder_verifyFareAboveTwoKms" time="1.238" classname="lm.lmSolution.PlaceOrderTests"/>
</testsuite> <!-- lm.lmSolution.PlaceOrderTests -->
